webrtc.html

```diff
@@ -8033,17 +8035,84 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
                 <p>The transport over which all SCTP packets for data channels
                 will be sent and received.</p>
               </dd>
+              <dt><dfn><code>state</code></dfn> of type <span class=
+              "idlAttrType"><a>RTCSctpTransportState</a></span>, readonly</dt>
+              <dd>
+                <p>The current state of the SCTP transport. The attribute returns
+                the value of the <dfn>[[\SctpTransportState]]</dfn> internal slot.</p>
+              </dd>
               <dt><dfn><code>maxMessageSize</code></dfn> of type <span class=
               "idlAttrType"><a>unsigned long</a></span>, readonly</dt>
               <dd>
                 <p>The maximum size of data that can be passed to
                 <code><a>RTCDataChannel</a></code>'s <code><a data-for=
                 "RTCDataChannel">send()</a></code> method.</p>
               </dd>
+              <dt><dfn><code>onstatechange</code></dfn> of type <span class=
+              "idlAttrType"><a>EventHandler</a></span></dt>
+              <dd>
+                <p>This event handler, of event handler event type
+                <code><a>statechange</a></code>, <em class="rfc2119" title="MUST">MUST</em>
+                be fired any time the <code><a>RTCSctpTransportState</a></code>
+                changes.</p>
+              </dd>
             </dl>
           </section>
         </div>
       </section>
+      <section id="rtcsctptransportstate*">
+        <h3><dfn>RTCSctpTransportState</dfn> Enum</h3>
+        <p><code>RTCSctpTransportState</code> indicates the state of the SCTP
+        transport.</p>
+        <div>
+          <pre class="idl">enum RTCSctpTransportState {
+    "new",
```

Comment:
I'm not sure we need "new", since we decided the RTCSctpTransport is only created on applying an answer, in which case it already has enough information to connect? Though maybe we should revisit that decision.

Reply:
There might be an advantage to having the `RTCSctpTransport` created earlier, in which case the "new" state might be used. But I agree that if it is only created on applying an answer it should be initialized to "connecting".

Reply:
Opening another issue relating to the transition text.